stake-pool/js/src/index.ts

```diff
@@ -312,7 +312,8 @@ export async function withdrawStake(
   voteAccountAddress?: PublicKey,
   stakeReceiver?: PublicKey,
   poolTokenAccount?: PublicKey,
-  validatorComparator?: (_a: ValidatorAccount, _b: ValidatorAccount) => number,
+  validatorComparator?: (a: ValidatorAccount, b: ValidatorAccount) => number,
+  validatorLimiter?: (a: ValidatorAccount) => number,
```

Comment:
Looks like the changes in this function also got pulled in accidentally

Reply:
will do another PR